Update Hypershift to v0.4 (#2166)
timescale/docs: cloud/migrate-to-cloud/hypershift.md

    accordingly. For example:

    ```yml
    source: "host=<DB_NAME> dbname=<SOURCE_DB_HOSTNAME> user=postgres port=5431"
    target: "host=<DB_NAME> dbname=<TARGET_DB_HOSTNAME> user=postgres port=5432"
    verbose: true
    hypertable_configs:
      - schema: public
        name: stocks_real_time
        time_column_name: time
        chunk_time_interval: "12h"
        compress:
          after: "48h"
          segmentby:
            - symbol
          orderby:
            - time desc
    ```

1.  At the command prompt, run the Hypershift container. Include the source and

>     destination database passwords, and the path to your `hypershift.yml`

Reviewer: @alejandrodnm I don't understand the value of `Include the source and destination database passwords`. The YAML file includes the source and destination DB_URI mentions, I would suggest you to recommend users using the YAML fields to setup DB_URI (along with passwords). This helps to:
1. Reduce the steps to docs. 
2. Doesn't split the config between YAML and CLI options by default. 

Of course, we will add the possibility of configuring DB_URI's in an other file, which contains all details of YAML and CLI options.

Reviewer 2: I think the way the command works has changed since I wrote this. If you can give me the correct command, I'll update the wording appropriately.

Author: I think storing passwords in a text file is a bad practice. It might lead to people forgetting to delete them and then they might leak.

    configuration file:

    ```bash
    docker run -v $(pwd)/hypershift.yml:/hypershift.yml \

>     -ti timescale/hypershift:0.4 clone \

Reviewer 3: Opened a PR for the same https://github.com/timescale/docs/pull/2201

    accordingly. For example:

    ```yml
    source: "host=<DB_NAME> dbname=<SOURCE_DB_HOSTNAME> user=postgres port=5431"
    target: "host=<DB_NAME> dbname=<TARGET_DB_HOSTNAME> user=postgres port=5432"
    verbose: true
    hypertable_configs:
      - schema: public
        name: stocks_real_time
        time_column_name: time
        chunk_time_interval: "12h"
        compress:
          after: "48h"
          segmentby:
            - symbol
          orderby:
            - time desc
    ```

1.  At the command prompt, run the Hypershift container. Include the source and
    destination database passwords, and the path to your `hypershift.yml`
    configuration file:

    ```bash
    docker run -v $(pwd)/hypershift.yml:/hypershift.yml \
    -ti timescale/hypershift:0.4 clone \
    -s "<DB_PASSWORD>" \
    -t "<DB_PASSWORD>" \
    --config-file /hypershift.yml
    ```

1.  When the migration has finished, a summary of the migration is shown, and
    the Docker container exits.

</Procedure>

> If you want to view the available configuration options for Hypershift, you
> can access its help by running the following command in your terminal:

Reviewer 2: ```suggestion
To see all available configuration options for Hypershift, use this command:
```
